lib/jsdom/browser/default-stylesheet.js

```diff
@@ -128,14 +123,6 @@ q {
     display: inline
 }
 
-q:before {
-    content: open-quote;
```

Comment:
Why are these removed?

Reply:
@domenic I have added the case insensitive flag for attributes, the `i` flag from the Selectors Level 4 specs have a look at current branch, also the new `nwsapi`will be ready to ship sometime next week with all the bells a whistles, I will also be able to port `:matches` to current branch.